seqr/management/tests/check_bam_cram_paths_tests.py

```diff
@@ -3,32 +3,53 @@
 from django.core.management import call_command
 from django.test import TestCase
 
+from seqr.models import IgvSample
+
 
 class CheckBamCramPathsTest(TestCase):
     fixtures = ['users', '1kg_project']
 
+    def setUp(self):
+        existing_sample = IgvSample.objects.first()
+        IgvSample.objects.create(
+            individual=existing_sample.individual,
+            sample_type=IgvSample.SAMPLE_TYPE_GCNV,
+            file_path='gs://missing-bucket/missing_file',
+        )
+
+    @mock.patch('hail.hadoop_is_file')
     @mock.patch('seqr.management.commands.check_bam_cram_paths.logger')
-    @mock.patch('seqr.views.utils.dataset_utils.validate_alignment_dataset_path')
-    def test_command(self, mock_validate_path, mock_logger):
+    def test_command_with_project(self, mock_logger, mock_hadoop_is_file):
+        mock_hadoop_is_file.return_value = False
         call_command('check_bam_cram_paths', '1kg project n\u00e5me with uni\u00e7\u00f8de')
+        self._check_results(1, mock_logger, mock_hadoop_is_file)
```

Comment:
can you also add a test that calling it with a different project doesn't check this sample?